.github/workflows/rust.yml

```diff
@@ -73,40 +72,16 @@ jobs:
       - uses: taiki-e/install-action@nextest
       - name: cargo test
         run: |
-          cargo nextest run --profile ci -E 'not package(~narwhal)'
+          cargo nextest run --profile ci
+      - name: narwhal celo crypto tests
+        run: cargo nextest run --features celo --profile ci -p narwhal-crypto
       - name: Doctests
         run: |
           cargo test --doc
```

Comment:
Let's also ensure that the docs tests will run as `cargo test --doc --features celo`